docs/GuiTestDocumentation.md

```diff
@@ -0,0 +1,282 @@
+| ![BeforeDeleteEvent](images/GUITestImages/BeforeDeleteEvent.png) | ![AfterDeleteEvent.png](images/GUITestImages/AfterDeleteEvent.png) |
+
+As you can see in the above images `Dione's Bday` was deleted with the appropriate command result message displayed in the CommandResult box.
+
+If incorrect format of `deleteevent` command is entered, the appropriate delete event error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![DeleteEventError](images/GUITestImages/DeleteEventError.png)
+
+#### Show all events
+
+Upon successful execution of `showfriends` command the following must occur
+
+1. Events tab is selected and displayed
+2. All the Events in Amigos are displayed
+
+|                             Before                             |                            After                             |
+|:--------------------------------------------------------------:|:------------------------------------------------------------:|
+| ![BeforeShowEvents](images/GUITestImages/BeforeShowEvents.png) | ![AfterShowEvents](images/GUITestImages/AfterShowEvents.png) |
+
+#### Find an event
+
+Upon successful execution of `findevent` command the following must occur
+
+1. Correct Command result display message
+2. Any friends who match the search criteria are displayed
+
+|                            Before                            |                           After                            |
+|:------------------------------------------------------------:|:----------------------------------------------------------:|
+| ![BeforeFindEvent](images/GUITestImages/BeforeFindEvent.png) | ![AfterFindEvent](images/GUITestImages/AfterFindEvent.png) |
+
+As you can see in the above images `findfriend n/Naa` was executed and since `Naaman's Bday` matches, that event card is displayed.
+
+If incorrect format of `findevent` command is entered, the appropriate find event error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![FindEventError](images/GUITestImages/FindEventError.png)
+
+### Logs
+
+Do note that after the execution of any log related commands, the GUI should switch to the `Friends` tab
+
+#### Add Log
+
+Upon successful adding of a log
+
+1. Correct Command result display message
+2. Log should be added and title should be displayed in the friend's card
+
+|                         Before                         |                        After                         |
+|:------------------------------------------------------:|:----------------------------------------------------:|
+| ![BeforeAddLog](images/GUITestImages/BeforeAddLog.png) | ![AfterAddLog](images/GUITestImages/AfterAddLog.png) |
+
+As you can see in the above images `WL's Bday` was inserted in the correct order with the appropriate command result message displayed in the CommandResult box.
```

Comment:
i think the description here is meant for `addlog` instead of the event WL's Bday!

Reply:
Thank for spotting this!